app/controllers/hyrax/workflow_actions_controller.rb

```diff
@@ -20,7 +20,7 @@ def update
 
     def curation_concern
       @curation_concern ||=
-        Hyrax.query_service.find_by_alternate_identifier(alternate_identifier: params[:id])
+        Hyrax::ValkyrieCanCanAdapter.find(nil, params[:id])
```

Comment:
should it be `load_resource` instead? or what's the right way to do this with CanCanCan directly, instead of hard coding the adapter?